### PR TITLE
[docs] - Implementing re-usable Markdown 

### PR DESCRIPTION
**Pull request by erinkcochran87**
This PR (attempts) to implement re-usable Markdown content via components.

### CODE REVIEW

**Comment by erinkcochran87** [comment]
@yuhan - I followed your PR for the blog (https://github.com/dagster-io/dagster-website/pull/136) to implement this, but am stuck. 

And am getting this console error:

```
ReferenceError: require is not defined
    at eval (eval at <anonymous> (/Users/erincochran/Desktop/dagster-repo/docs/next/node_modules/next-mdx-remote/render-to-string.js:41:14), <anonymous>:5:25)
    at /Users/erincochran/Desktop/dagster-repo/docs/next/node_modules/next-mdx-remote/render-to-string.js:51:8
    at async getStaticProps (webpack-internal:///./pages/[...page].tsx:269:27)
    at async Object.renderToHTML (/Users/erincochran/Desktop/dagster-repo/docs/next/node_modules/next/dist/server/render.js:465:20)
    at async doRender (/Users/erincochran/Desktop/dagster-repo/docs/next/node_modules/next/dist/server/base-server.js:879:38)
    at async cacheEntry.responseCache.get.isManualRevalidate.isManualRevalidate (/Users/erincochran/Desktop/dagster-repo/docs/next/node_modules/next/dist/server/base-server.js:979:28)
    at async /Users/erincochran/Desktop/dagster-repo/docs/next/node_modules/next/dist/server/response-cache.js:64:36
```

File: docs/content/getting-started.mdx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Welcome to Dagster!
 
+<OpenUserSettings />
```
Comment (yuhan) [inline docs/content/getting-started.mdx:3]: [1]

File: docs/next/components/mdx/MDXComponents.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ import "react-medium-image-zoom/dist/styles.css";
 export const SearchIndexContext = React.createContext(null);
 import path from "path";
 import { Transition } from "@headlessui/react";
+import OpenUserSettings from "./OpenUserSettings.mdx";
```
Comment (yuhan) [inline docs/next/components/mdx/MDXComponents.tsx:19]: [1]

File: docs/next/components/mdx/MDXComponents.tsx [hunk 3]
```diff
@@ -448,4 +449,5 @@ export default {
   PlaceholderImage,
   Experimental,
   Icons,
+  OpenUserSettings,
```
Comment (yuhan) [inline docs/next/components/mdx/MDXComponents.tsx:452]: [1]

**Review by yuhan — COMMENTED** [review]
you'll need to include the mdx as a component, i.e. [1]

**Review by yuhan — APPROVED** [review]

**Pull request merged**